index.js

```diff
@@ -1,24 +1,30 @@
 'use strict';
 const UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 
-module.exports = num => {
+module.exports = (num, options) => {
 	if (!Number.isFinite(num)) {
 		throw new TypeError(`Expected a finite number, got ${typeof num}: ${num}`);
 	}
 
+	const signed = options && options.signed;
+	if (signed && num === 0) {
+		return '±0 B';
```

Comment:
Why does this have a sign? I think it should be just `0 B`.

Reply:
I think if positive and negative numbers have signs then zero should have one too. It also looks better aligned when you have a list of numbers.

Reply:
I don't think it's even correct sign to use: https://en.m.wikipedia.org/wiki/Plus-minus_sign

I would just pad it with a space.

Reply:
hmm, that's a good point 🤔 

should we pad with space ourselves or make it the responsibility of the user?

Reply:
I think we should pad it. It's easy for the user to just do `.trim()` if they don't want it, but harder to add if they do.